cmd/dlb_plugin/README.md

```diff
@@ -19,6 +19,137 @@ Table of Contents
 
 This Intel DLB device plugin provides support for [Intel DLB](https://builders.intel.com/docs/networkbuilders/SKU-343247-001US-queue-management-and-load-balancing-on-intel-architecture.pdf) devices under Kubernetes.
 
+### DLB2 driver configuration
+
+You should first build and load the dlb2 driver. Get [DLB software release](https://downloadmirror.intel.com/690271/dlblinuxsrcrelease7.4.020211006.txz), build the kernel module and load the module as follows:
+```bash
+$ wget -q https://downloadmirror.intel.com/690271/dlblinuxsrcrelease7.4.020211006.txz -O- | tar -Jx
+$ cd ./dlb/driver/dlb2/ && make
+$ sudo insmod dlb2.ko
+```
+
+Then, available dlb device nodes are visible in devfs.
+```bash
+$ ls -1 /dev/dlb*
+/dev/dlb0  /dev/dlb1  /dev/dlb2 ...
+```
+
+Run libdlb example apps:
+```bash
+$ cd ../../libdlb/ && make && sudo LD_LIBRARY_PATH=$PWD ./examples/dir_traffic -n 128
+/dev/dlb0  /dev/dlb1  /dev/dlb2 ...
+```
+
+### DPDK PF configuration
+Install build dependencies:
+> **Note:**: This configuration is based on Ubuntu/Debian distribution. For other distributions that do not use apt, install the dependencies using another way.
+```bash
+$ sudo apt-get update && sudo apt-get install build-essential meson python3-pyelftools libnuma-dev python3-pip
+sudo pip install ninja
+```
+
+Patch dpdk sources to work with DLB and build dpdk:
+```bash
+$ wget -q https://fast.dpdk.org/rel/dpdk-20.11.3.tar.xz -O- | tar -Jx
+$ wget -q https://downloadmirror.intel.com/690271/dlblinuxsrcrelease7.4.020211006.txz -O- | tar -Jx
+$ cd ./dpdk-stable-20.11.3 && patch -p1 < ../dlb/dpdk/dpdk_dlb_v20.11.3_29751a4_diff.patch
+$ sed -i 's/270b,2710,2714/270b,2710,2711,2714/g' ./usertools/dpdk-devbind.py
+$ meson setup --prefix $(pwd)/installdir builddir && ninja -C builddir install
+```
+
+Load vfio-pci kernel module:
+```bash
+$ sudo modprobe vfio-pci
+```
+
+Bind DLB device(s) to vfio-pci driver:
+```bash
+$ sudo ./usertools/dpdk-devbind.py --unbind 0000:6d:00.0 && sudo ./usertools/dpdk-devbind.py --bind vfio-pci 0000:6d:00.0
+```
+
+Make sure that DLB device is in the list of eventdev devices known to dpdk and handled by vfio-pci driver:
+```bash
+$ sudo ./usertools/dpdk-devbind.py --status | grep -A3 'Eventdev devices using DPDK-compatible driver'
+```
+
+Run eventdev test for the PF device:
+```bash
+$ sudo ./builddir/app/dpdk-test-eventdev --no-huge -a '6d:00.0' -- --test=order_queue --plcores=1 --wlcore=2,3 --nb_flows=64
+```
+
+> **Note:**: Alternative and faster way is to use Dockerfile from this [PR](https://github.com/intel/intel-device-plugins-for-kubernetes/pull/733) for patching and building DPDK and running tests.
```

Comment:
```suggestion
> **Note:**: Alternative and faster way is to use [Dockerfile](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/demo/dlb-dpdk-demo/Dockerfile) for patching and building DPDK and running tests.

Reply:
This is enough for now and we don't need to add anything to `demo/readme.md`.